convert/string.go

```diff
@@ -9,16 +9,19 @@ import (
 )
 
 func StringToTime(s string) *time.Time {
+	t := new(time.Time)
 	if s == "" {
-		return nil
+		return t
 	}
+
 	i, err := strconv.ParseInt(s, 10, 64)
 	if err != nil {
 		fmt.Println("Error:", err)
 		os.Exit(1)
 	}
-	t := time.Unix(i, 0)
-	return &t
```

Comment:
Is it best practice to avoid returning `&t`? I'm still new to GoLang and want to make sure I'm doing a better job leveraging the language.

Reply:
Pointers tend to add complexity so it's better to avoid them unless you need them. In this case I'd just return `time.Time`.